dali/pipeline/operators/util/optical_flow.h

```diff
@@ -80,11 +82,52 @@ class OpticalFlow : public Operator<Backend> {
 
 
  private:
+  /**
+   * Optical flow lazy initialization
+   */
+  void of_lazy_init(size_t width, size_t height, size_t channels, cudaStream_t stream) {
+    std::call_once(of_initialized_,
+                   [&]() {
```

Comment:
here just a personal preference comment. I think that those indents are too big. Horizontal space is precious :)

Reply:
Yeah, that's the tradeoff: small indents or seeing lambda clearly ;)